packages/story-editor/src/components/floatingMenu/elements/shared/useUpdateSelected.js

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import { useCallback } from '@googleforcreators/react';
+
+/**
+ * Internal dependencies
+ */
+import { useStory } from '../../../../app';
+
+function useUpdateSelected() {
+  const { updateSelectedElements } = useStory(
+    ({ actions: { updateSelectedElements } }) => ({ updateSelectedElements })
+  );
```

Comment:
nit: destructuring this is redundant

```suggestion
  const updateSelectedElements = useStory(({ actions: { updateSelectedElements } }) => updateSelectedElements);
```

Reply:
Also with the default identify function being `shallowEqual`? I hadn't actually tried this, but just kind of assumed that we could only select an object of things, not a single thing, while still having memoization work correctly.

Reply:
But if that works, this hook is definitely pretty useless, as it was mainly to get around this slightly annoying restriction.

Also, we should do that throughout the codebase - I think we have potentially hundreds of these spread throughout the codebase.

Reply:
Seems to work great. Optimized this in fbdaaec8ba.